src/react-apps/applications/service-development/src/features/deploy/components/deployPaper.tsx

```diff
@@ -0,0 +1,369 @@
+/* tslint:disable:max-line-length */
+
+import { Grid, Paper, Typography } from '@material-ui/core';
+import { createMuiTheme, createStyles, withStyles } from '@material-ui/core/styles';
+import { TypographyProps } from '@material-ui/core/Typography';
+import classNames from 'classnames';
+import * as React from 'react';
+import AltinnButton from '../../../../../shared/src/components/AltinnButton';
+import AltinnIcon from '../../../../../shared/src/components/AltinnIcon';
+// import AltinnSpinner from '../../../../../shared/src/components/AltinnSpinner';
```

Comment:
Remove comment if it's not going to be used.

Reply:
I will. It will be used. That part is not completed yet.